README.md

```diff
@@ -113,14 +94,10 @@ class MyElement extends Polymer.Element {
 Define a Polymer property.
 
 `options` is a [Polymer property
-options](https://www.polymer-project.org/2.0/docs/devguide/properties) object.
+options](https://www.polymer-project.org/3.0/docs/devguide/properties) object.
 All standard options are supported, except for `value`; use a property
-initializer instead.
-
-If the [Metadata Reflection API](#metadata-reflection-api) is configured, the
-`type` option (which determines how Polymer de-serializes Element attributes
-for this property) will be inferred from the TypeScript type and can be
-omitted.
+initializer instead. `type` is required, and must be one of the Polymer property
+constructor types (`String`, `Object`, etc.).
```

Comment:
Polymer itself defaults the type to String, should we?

Reply:
I feel like it's pretty easy to forget, especially for users who may previously have been relying on reflect metadata, who would now just silently get a different de-serialization behavior.